source/framework/core/inc/TRestHits.h

```diff
@@ -21,40 +21,32 @@
 #ifndef TRestSoft_TRestHits
 #define TRestSoft_TRestHits
 
+#include <TArrayD.h>
+#include <TArrayI.h>
 #include <TCanvas.h>
 #include <TF1.h>
 #include <TGraphErrors.h>
 #include <TH1.h>
+#include <TMath.h>
+#include <TMatrixD.h>
 #include <TVector3.h>
 
 #include <iostream>
 
-#include "TArrayD.h"
-#include "TArrayI.h"
-#include "TCanvas.h"
-#include "TMath.h"
-#include "TMatrixD.h"
-#include "TObject.h"
-
 enum REST_HitType { unknown = -1, X = 2, Y = 3, Z = 5, XY = 6, XZ = 10, YZ = 15, XYZ = 30 };
-//! It let save an event as a set of punctual deposition.
+//! Allows saving an event as a set of punctual deposition.
 //! It saves a 3-coordinate position and an energy for each punctual deposition.
-class TRestHits : public TObject {
```

Comment:
Do you we have to increase the library version because we remove the inheritance from `TObject`? I am not sure of the backward compatibility in this case...

Reply:
I was thinking no but I am not sure to be honest. I will increase it just in case.